src/transport/PairingSession.h

```diff
@@ -177,10 +177,9 @@ class DLL_EXPORT PairingSession
     // TODO: remove Clear, we should create a new instance instead reset the old instance.
     void Clear()
     {
-        mSecureSessionType = Transport::SecureSession::Type::kUndefined;
-        mPeerNodeId        = kUndefinedNodeId;
-        mPeerCATs          = kUndefinedCATs;
-        mPeerAddress       = Transport::PeerAddress::Uninitialized();
+        mPeerNodeId  = kUndefinedNodeId;
```

Comment:
```suggestion
        // NOTE: no need to clear mSecureSessionType as it cannot change.
        mPeerNodeId  = kUndefinedNodeId;
```

Reply:
Might even be better, to make mSecureSessionType const, initialize it from an arg to the constructor, and have derived classes call base class constructor with proper arg. Then it always reflects actual object and can't be altered.

Reply:
Updated. Made mSecureSessionType const.